projects/telemetry/src/nmea.c

```diff
@@ -0,0 +1,33 @@
+#include "nmea.h"
+#include <stdbool.h>
+#include <string.h>
+
+static char s_hex[] = "0123456789ABCDEF";
+
+// out must be a pointer to at least 3 chars
+void prv_int_to_hex(uint8_t checksum, char *out) {
+  // we know that it'll be a 2 digit hex number since the checksum is 8-bit
+  out[0] = s_hex[checksum >> 4];
+  out[1] = s_hex[checksum & 15];
```

Comment:
Can you use hex instead? Using decimal values for bitwise operations feels a little odd. It's hard to translate in your head.

Reply:
Yes, that would be easier to understand.